frontend/src/styles/settings/variables.scss

```diff
@@ -59,6 +59,9 @@ $pagination-item-hover: darken($violet, 20%);
 $pagination-item-border-hover: $pagination-item-hover;
 $card-border-color: $violet;
 
+//Backgrounds
+$secondary-background-colour: $lightest-grey;
```

Comment:
I'm not sure that we need two separate `// Backgrounds` sections. And I'm open to input here (cc: @mmumby), but I don't think we need to alias our background colours

If we are going to do that, could you do a quick find-and-replace of the current uses of `$lightest-grey` with your alias? I used `$lightest-grey` in a couple of places already, and having two references to the same background colour defeats the purpose of the variables

Reply:
I only did this as it was suggested that I should by @mmumby, I can remove it if necessary.

Reply:
Yeah, I did suggest this to him as he is using it as the background for a smaller div, and if the background color changes I didn't want that div to get forgotten. 

I like Nathans suggestion to replace the other background colors with this variable.

Reply:
All have been replaced.